app/javascript/components/zone-form/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './zone-form.schema';
+
+const ZoneForm = ({recordId}) => {
+    const [initialValues, setInitialValues] = useState([]);
```

Comment:
The `initialValues` is expected to be an object, so an array might cause issues when not in edit mode. Can you change it to an empty object?